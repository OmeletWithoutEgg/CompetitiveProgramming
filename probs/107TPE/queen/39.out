559546.11 5912102.80
