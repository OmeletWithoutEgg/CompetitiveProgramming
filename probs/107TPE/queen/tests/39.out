471909.00 -48710955.00
